src/components/VaccinationsTable/utils.ts

```diff
@@ -0,0 +1,71 @@
+import regions, { Region } from 'common/regions';
+import orderBy from 'lodash/orderBy';
+import take from 'lodash/take';
+import takeRight from 'lodash/takeRight';
+import { MapView } from 'screens/HomePage/HomePage';
+import { LocationSummariesByFIPS } from 'common/location_summaries';
+import { Metric } from 'common/metricEnum';
+
+export interface RegionVaccinationInfo {
+  regionName: string;
+  vaccinationsInitiated: number;
+  vaccinationsCompleted: number;
+  rank: number;
+  url: string;
+}
+
+function getRegionsSortedByVaccinationsInitiated(
+  regionScope: MapView,
+): RegionVaccinationInfo[] {
+  const regionsToSort =
+    regionScope === MapView.COUNTIES ? regions.counties : regions.states;
+
+  const regionsWithVaccinationMetrics = regionsToSort.map((region: Region) => {
+    const summaryForFips = LocationSummariesByFIPS[region.fipsCode];
+    return {
+      regionName: region.name,
+      vaccinationsInitiated:
+        summaryForFips?.metrics[Metric.VACCINATIONS]?.value ?? null,
+      vaccinationsCompleted: summaryForFips?.vc ?? null,
+      url: region.canonicalUrl,
```

Comment:
nit: not sure if it really makes any difference, but I think we use `relativeUrl` in other places (like the map) where we are just linking internally.